gokart/build.py

```diff
@@ -9,14 +9,18 @@
 from gokart.utils import check_config, read_environ
 
 
-class HideLogger:
-    def __init__(self, verbose: bool):
+class LoggerConfig:
+    def __init__(self, verbose: bool, level: Optional[int]):
         self.verbose = verbose
         self.logger = getLogger(__name__)
         self.default_level = self.logger.level
+        self.log_level_in_build = level
 
     def __enter__(self):
-        if not self.verbose:
+        if self.log_level_in_build is not None:
+            logging.disable(self.log_level_in_build)
+            self.logger.setLevel(self.log_level_in_build)
+        elif not self.verbose:
```

Comment:
[help wanted]
In this implementation, `log_level_in_build` is prioritized than `verbose`.
Which do you prefer this or `verbose` is prioritized one?

Reply:
How do you think about remove `verbose`, and setting the default for `loglevel`?

```python
class LoggerConfig:
    def __init__(self, level: Optional[int]=logging.CRITICAL):
    ....

    def __enter__(self):
        if self.level==logging.CRITICAL:
            logging.disable(sys.maxsize)
        else:
            logging.disable(self.level)
        self.logger.setLevel(self.level)
        return self
```

Reply:
Backward compatibility will be lost, though. 
If you don't want to do that, I agree with @hirosassa 's  implementation.

Reply:
I'm standing with @vaaaaanquish 's suggestion. Maintaining both verbose and level options are confusing to me.

Reply:
I'm standing with @vaaaaanquish 's suggestion. Maintaining both verbose and level options are confusing to me.

Reply:
@vaaaaanquish @Hi-king Thanks for your fruitful FBs. fixed!